serveo/run.sh

```diff
@@ -45,7 +45,7 @@ then
     PORT3=" -R  ${DOMAIN}:${PORT3TO}:localhost:${PORT3FROM}"
 fi
 
-CMD="/bin/bash -c 'sleep ${RETRY_TIME} && ssh ${IDENTITY} -tt -o ExitOnForwardFailure=yes -o StrictHostKeyChecking=no -o ServerAliveInterval=10 -o ServerAliveCountMax=3 ${PORT1}${PORT2}${PORT3} ${ALIAS}@${SERVER}'"
+CMD="/bin/bash -c 'sleep ${RETRY_TIME} && ssh ${IDENTITY} -tt -o ExitOnForwardFailure=yes -o StrictHostKeyChecking=no -o ServerAliveInterval=10 -o ServerAliveCountMax=3 ${PORT1}${PORT2}${PORT3} -p ${SSH_PORT} ${ALIAS}@${SERVER}'"
```

Comment:
I added it as an optional parameter thinking that it might break for those who are using old configurations(without `ssh_port` parameter), after the update. Does the configuration gets updated on addon update? :thinking:

Reply:
Hm... that's a good question, I've took it for granted - with default values placed in our `config.json` but I need to verify it on my instance...